### DISCUSSION TITLE
How to disable X-Powered-By and Server-Timing headers?

### DISCUSSION THREAD

**Discussion by rudimadima** [nuxt/nuxt#16877] [Questions]
I have 2 additional headers in my Nuxt 3 website answer:
```
X-Powered-By: Nuxt
Server-Timing: -;dur=0;desc="Generate"
```
How to disable them?
In nuxt.config.ts Typescript shows that timing option is deprecated.

**Comment by TheAlexLichter** [comment]
You could use a nitro middleware (also called server middleware) and set headers ☺️
